MODULE.bazel

```diff
@@ -3,16 +3,59 @@ module(
     version = "1.37.0-dev",
 )
 
+# All bazel_dep entries are organized alphabetically for improved readability.
+# Runtime dependencies from Bazel Central Registry
+bazel_dep(name = "apple_support", version = "1.23.1", repo_name = "build_bazel_apple_support")
 bazel_dep(name = "aspect_bazel_lib", version = "2.21.2")
+bazel_dep(name = "aspect_rules_js", version = "2.5.0")
+bazel_dep(name = "bazel_features", version = "1.38.0")
+bazel_dep(name = "bazel_skylib", version = "1.8.2")
+bazel_dep(name = "boringssl", version = "0.20250514.0")
+bazel_dep(name = "brotli", version = "1.1.0", repo_name = "org_brotli")
+bazel_dep(name = "cel-spec", version = "0.24.0", repo_name = "dev_cel")
+bazel_dep(name = "emsdk", version = "4.0.13")
 bazel_dep(name = "envoy_api", version = "1.37.0-dev")
 bazel_dep(name = "envoy_build_config", version = "1.37.0-dev")
 bazel_dep(name = "envoy_mobile", version = "1.37.0-dev")
+bazel_dep(name = "fast_float", version = "8.0.2")
+bazel_dep(name = "flatbuffers", version = "25.2.10", repo_name = "com_github_google_flatbuffers")
+bazel_dep(name = "fmt", version = "12.1.0", repo_name = "com_github_fmtlib_fmt")
+bazel_dep(name = "gazelle", version = "0.47.0", repo_name = "bazel_gazelle")
+bazel_dep(name = "googleapis", version = "0.0.0-20241220-5e258e33.bcr.1", repo_name = "com_google_googleapis")
+bazel_dep(name = "googleurl", repo_name = "com_googlesource_googleurl")
```

Comment:
Needs version. Current version at `dd4080fec0b443296c0ed0036e1e776df8813aa7`

Reply:
I removed from the scope of this PR